demos/client-frictionless-example/src/App.tsx

```diff
@@ -29,6 +29,7 @@ function App() {
             (process.env.PROSOPO_DEFAULT_ENVIRONMENT as EnvironmentTypes) || EnvironmentTypesSchema.enum.development,
         serverUrl: process.env.PROSOPO_SERVER_URL || '',
         atlasUri: process.env._DEV_ONLY_WATCH_EVENTS === 'true' || false,
+        createHoneypot: process.env.PROSOPO_HONEYPOT === 'true',
```

Comment:
Why is this here if there is a `getHoneypot` function?

Reply:
Okay, I will check this again and if it doesn't need, I will remove this